I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 61, in script
    $ logos += 1
  File "game/script.rpy", line 61, in <module>
    $ logos += 1
NameError: name 'logos' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 61, in script
    $ logos += 1
  File "C:\Users\mirna\Documents\Ren'py\renpy-7.3.5-sdk\renpy\ast.py", line 914, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\mirna\Documents\Ren'py\renpy-7.3.5-sdk\renpy\python.py", line 2028, in py_exec_bytecode
    exec bytecode in globals, locals
  File "game/script.rpy", line 61, in <module>
    $ logos += 1
NameError: name 'logos' is not defined

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Fountain of youth 1.0
Tue Jan 12 18:49:59 2021
